lib/test-service-provider.js

```diff
@@ -64,15 +72,17 @@
 				return service;
 			}
 
-			// Return a real registered service, if it exists (sometimes you want the real deal, i.e. utils)
-			preRegisteredService = application.getServiceForTest(serviceName, this);
-			if (preRegisteredService) {
-				return preRegisteredService;
+			if (this.allowedServices.hasOwnProperty(serviceName)) {
```

Comment:
I might be reading this wrong, but I thought the whitelist was optional so if it wasn't present, all services were allowed? This seems to require all pass through services to be defined or else it errors. Is that intentional?